service/src/test/java/greencity/service/HabitAssignServiceImplTest.java

```diff
@@ -387,6 +395,28 @@ void getAllHabitAssignsByUserIdAndStatusNotCancelled() {
         assertEquals(habitAssignDtos, actual);
     }
 
+    @Test
+    void getUserAndUserCustomShoppingList() {
+        List<CustomShoppingListItemResponseDto> customShoppingListItemResponseDtos =
+            Collections.singletonList(customShoppingListItemResponseDto());
+        List<UserShoppingListItemResponseDto> userShoppingListItemResponseDtos =
+            Collections.singletonList(getCustomUserShoppingListItemDto());
+        HabitAssignUserAndUserCustomShoppingListDto expected = HabitAssignUserAndUserCustomShoppingListDto
+            .builder()
+            .customShoppingListItemDto(customShoppingListItemResponseDtos)
+            .userShoppingListItemsDto(userShoppingListItemResponseDtos)
+            .build();
+
+        when(shoppingListItemService.getUserShoppingList(anyLong(), eq(1L), eq("en")))
```

Comment:
It is better to create `useId`, `habitId` and `language` variables and use them when you mock some behavior